pom.xml

```diff
@@ -31,15 +31,12 @@
     <properties>
         <revision>2.0.3</revision>
         <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.60.3</jenkins.version>
+        <jenkins.version>2.164.3</jenkins.version>
```

Comment:
the jackson plugin requires 2.138, and its almost impossible to install a plugin while doing interactive testing when the version is pinned like this, i.e. pipeline can't be installed to verify that pipeline works.

Note: the minimum recommended version is currently 2.164 (2.150 on the page but 2.164 is the oldest maintained update center line) as documented here:
https://jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/#choosing-a-version